gradle.properties

```diff
@@ -43,3 +43,7 @@ kotlin.native.useEmbeddableCompilerJar=true
 kotlin.mpp.androidSourceSetLayoutVersion=2
 # Enable kotlin/native experimental memory model
 kotlin.native.binary.memoryModel=experimental
+
+# Gradle will show a warning when a Kotlin/Native target cannot be built on this machine.
+# Disable such message.
+kotlin.native.ignoreDisabledTargets=true
```

Comment:
This is used to remove an annoying warning message while building through gradle.  The message is about unsopported platforms(ie mac/ios) not being able to be build.